INSTALL.md

```diff
@@ -609,6 +609,32 @@ a terminal in order to make all required tools available via the command line:
 
      xcode-select --install
 
+You might want to consider using Readline by installing it via:
```

Comment:
```suggestion
You might want to consider using GNU readline by installing it via:
```
There are several readline implementation; macOS (more generally: BSD based systems) has its own `libreadline`, so let's be specific here.